src/Terminal/DirectoryHelpers.ts

```diff
@@ -316,3 +319,20 @@ export function areImportsEquals(f0: string, f1: string): boolean {
   if (!f1.endsWith(".ns") && !f1.endsWith(".js")) f1 = f1 + ".js";
   return areFilesEqual(f0, f1);
 }
+
+export function resolveImportPath(parentScriptPath: string, importedScriptPath: string): string {
```

Comment:
This could use an doc comment describing what the variables are. What does path.dirname do if the folder path is given rather than the full path to the script?